test/fluent-schema.test.js

```diff
@@ -84,13 +83,14 @@ test('use fluent-json-schema object', t => {
     query: { surname: 'bar' },
     payload: { name: 'foo' }
   }, (err, res) => {
-    t.error(err)
-    t.equal(res.statusCode, 200)
-    t.same(res.json(), { name: 'a', surname: 'b' })
+    t.assert.ifError(err)
+    t.assert.strictEqual(res.statusCode, 200)
+    t.assert.deepStrictEqual(res.json(), { name: 'a', surname: 'b' })
+    done()
```

Comment:
This `done` could resolve before the `inject` stuff has completed.

We should migrate this test to `async/await` for simplicity or the `done` must wait all the `inject` calls

Reply:
I migrated the "`use fluent-json-schema object`" test with `async/await`, (see commit here: https://github.com/fastify/fastify/pull/5832/commits/7a953975547840127ab723d5ad64c79e682fadf8)